examples/docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Run Remix With docker-compose
+
+Very simple example to run remix app with docker and docker-compose. 
+
+**With this example Remix can be run in any server have docker.**
```

Comment:
i don't know how to explain it but i will explain my use case:
if you have full control on server and want to run node js application you need to install nodejs and then you will need to test the the application in the new environment (like if you work on windows and the server is ubuntu or mac) so you need to spend time in this.
and after this you need to run application using PROCESS MANAGER like pm2,
all this can be solved by using docker. or this what i use docker for it